Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationObserver.cs

```diff
@@ -209,10 +193,14 @@ private void OnReceive(IncomingMessage data)
                             }
                         }
                         break;
-
-                    case ConnectedObserver.SpatialLocalizationMessageHeader:
-                        DebugLog("Passing message to observer");
-                        connectedObserver.ReceiveMessage(reader);
+                    default:
+                        if (CommandService.Instance.CommandHandlerDictionary.TryGetValue(command, out var handlers))
+                        {
+                            foreach (var handler in handlers)
+                            {
+                                handler.HandleCommand(data.Endpoint, command, reader);
+                            }
+                        }
```

Comment:
Maybe this method could just be a method on CommandService so that the logic for how to broadcast the command is contained there?